libs/zzz/ui/src/Character/card/CharacterCardEquipmentRow.tsx

```
@@ -0,0 +1,154 @@
+              <Box
+                component={NextImage ? NextImage : 'img'}
+                display="flex"
+                position="absolute"
+                src={
+                  '/@fs/D:/Coding/mygithub/genshin-optimizer/libs/zzz/ui/src/Character/card/discBackdrop.png'
+                }
+                width="41px"
+                height="36px"
+                top="2px"
+                left="6px"
+              ></Box>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)